package.json

```diff
@@ -9,7 +9,9 @@
   },
   "scripts": {
     "cpd": "jscpd ./packages/*/src",
-    "cpd:ci": "yarn cpd --silent --threshold 20"
+    "cpd:ci": "yarn cpd --silent --threshold 20",
+    "server:update": "cd packages/server && yarn server:update",
+    "server:deploy": "cd packages/server && yarn server:deploy"
```

Comment:
Do we still need these two here? I don't think they exist

Reply:
Nope, they do not. Probably need to remove